src/vdaf/prg.rs

```diff
@@ -48,12 +52,24 @@ impl<const SEED_SIZE: usize> Seed<SEED_SIZE> {
     }
 }
 
+impl<const SEED_SIZE: usize> Default for Seed<SEED_SIZE> {
+    fn default() -> Self {
+        Self([0u8; SEED_SIZE])
+    }
+}
+
 impl<const SEED_SIZE: usize> AsRef<[u8; SEED_SIZE]> for Seed<SEED_SIZE> {
     fn as_ref(&self) -> &[u8; SEED_SIZE] {
         &self.0
     }
 }
 
+impl<const SEED_SIZE: usize> AsMut<[u8]> for Seed<SEED_SIZE> {
```

Comment:
I understand that this exists to be used in conjunction with `Seed::default`. Are there risks to allowing mutation of the seed after it's initialized, or more to the point after some randomness has already been generated from it?

As an alternative to the `Seed::from_byte_iter` idea I sketched above, maybe what this calls for is a `SeedBuilder` struct, such that only `SeedBuilder` allows mutating the inner buffer and then after you `.build()` it into a `Seed`, all you can do is read the bytes.